From 4ab16114870a9d69f9813a60d047d1e94326322a Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:13 +0100
Subject: [PATCH 09/45] core: do not apply presets.

---
 src/core/manager.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 629966e..8e8ef5a 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -1636,8 +1636,6 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         if (r < 0)
                 return r;
 
-        manager_preset_all(m);
-
         lookup_paths_log(&m->lookup_paths);
 
         {
-- 
2.31.1

