From c398151c287b97724a0f98efb59fccc5e911341d Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Wed, 9 Jun 2021 10:33:14 -0300
Subject: [PATCH 25/45] Ensure /var/run is never a directory.

---
 src/shared/mount-setup.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/shared/mount-setup.c b/src/shared/mount-setup.c
index 9fec92c..01810d5 100644
--- a/src/shared/mount-setup.c
+++ b/src/shared/mount-setup.c
@@ -502,6 +502,13 @@ int mount_setup(bool loaded_policy, bool leave_propagation) {
         if (r < 0)
                 return r;
 
+        r = lstat("/var/run", &sb);
+        if (r == 0 && sb.st_mode & S_IFDIR) {
+                char path[256];
+                sprintf(path, "/var/run.%i", (int)time(NULL));
+                rename("/var/run", path);
+        }
+
 #if HAVE_SELINUX || ENABLE_SMACK
         /* Nodes in devtmpfs and /run need to be manually updated for
          * the appropriate labels, after mounting. The other virtual
-- 
2.31.1

